UI/src/app/components/comfy/comfy.component.ts

```diff
@@ -241,14 +240,15 @@ export class ComfyComponent extends BaseComponent
   initializeTargetData() {
     this.target = {};
 
-    const prefix = "target";
+    const prefix = 'target';
 
     this.state.target.tables.map(table => {
-      if(this.COLUMNS_TO_EXCLUDE_FROM_TARGET.findIndex(name => name === table.name)<0)
-      {this.target[table.name] = {};
-      this.target[table.name].name = `${prefix}-${table.name}`;
-      this.target[table.name].first = table.name;
-      this.target[table.name].data = [table.name];}
+      if (this.COLUMNS_TO_EXCLUDE_FROM_TARGET.findIndex(name => name === table.name) < 0) {
+        this.target[table.name] = {};
```

Comment:
could we also refactor this part to :
`
this.target[table.name]={
name: `${prefix}-${table.name}`,
first: table.name,
data: [table.name]
}
` 
?
Also table.name could be moved into const

Thank you!

Reply:
done